### ISSUE TITLE
[TW-1576] Invalid JSON output from export command : "urgency":H

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1602]
_Frédéric Meynadier on 2015-03-17T09:15:11Z says:_ 

 Using the "export" command to retrieve task data (json.array=on), some tasks come now with a field like this :

{code}
"urgency":H
{code}

... which makes the python 3.4 json parsing (json.loads()) fail. Most others have a numeric value here.

Up to version 2.4.0 the output for the same task was

{code}
"urgency":"H"
{code}
 
which seems reasonable, and works with the same json.load() function...

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-03-17T09:15:11Z
Modified: 2015-03-18T16:11:42Z
```

**Comment by taskwarrior** [comment]
_Tomas Babej on 2015-03-18T14:39:22Z says:_ 

 It works for me here:

{code}
task rc.json.array=on 69 export
[
{"id":69,"description":"Write HOW-TO use jenkins internally","entry":"20150223T170903Z","modified":"20150223T170903Z","priority":"H","project":"Work.Jenkins","status":"pending","uuid":"33f39ef8-7093-4984-85f7-fe47e974df13","urgency":7.12055}
]
Configuration override rc.json.array:on
{code}

Maybe something more than the information provided causes the misbehavior.

**Comment by taskwarrior** [comment]
_Frédéric Meynadier on 2015-03-18T16:08:16Z says:_ 

 Abstract : You're right. My DB was corrupted (but it did not appear before). Thanks ! I think this issue can be closed...

Here is the full export of one of the tasks :

{code}
[
{"id":0,"description":"corriger la tropo","end":"20140318T094135Z","entry":"20140219T164813Z","modified":"20140318T094135Z","project":"aces","status":"completed","urgency":H,"uuid":"278e5d73-fb9c-49a5-b85c-b92fc488ed8b","annotations":[{"entry":"20140221T165625Z","description":"Started task"},{"entry":"20140221T183150Z","description":"Stopped task"},{"entry":"20140224T153611Z","description":"Started task"},{"entry":"20140224T170459Z","description":"Stopped task"},{"entry":"20140224T170545Z","description":"Started task"},{"entry":"20140224T171258Z","description":"Stopped task"},{"entry":"20140225T110006Z","description":"Started task"},{"entry":"20140225T111156Z","description":"Stopped task"},{"entry":"20140225T130904Z","description":"Started task"},{"entry":"20140225T135417Z","description":"Stopped task"},{"entry":"20140225T144819Z","description":"Started task"},{"entry":"20140225T160249Z","description":"Stopped task"},{"entry":"20140226T161739Z","description":"Started task"},{"entry":"20140226T172046Z","description":"Stopped task"},{"entry":"20140227T133423Z","description":"Started task"},{"entry":"20140227T142139Z","description":"Stopped task"},{"entry":"20140304T143329Z","description":"Started task"},{"entry":"20140304T145000Z","description":"Stopped task"},{"entry":"20140304T172258Z","description":"Started task"},{"entry":"20140304T175943Z","description":"Stopped task"},{"entry":"20140305T080033Z","description":"Started task"},{"entry":"20140305T083036Z","description":"Stopped task"}],"urgency":4}
]
{code}

There are 2 "urgency" fields, one with value "H", the other with value "4". 

Here is the corresponding line in the "completed.data" file :

{code}
[annotation_1393001785:"Started task" annotation_1393007510:"Stopped task" annotation_1393256171:"Started task" annotation_1393261499:"Stopped task" annotation_1393261545:"Started task" annotation_1393261978:"Stopped task" annotation_1393326006:"Started task" annotation_1393326716:"Stopped task" annotation_1393333744:"Started task" annotation_1393336457:"Stopped task" annotation_1393339699:"Started task" annotation_1393344169:"Stopped task" annotation_1393431459:"Started task" annotation_1393435246:"Stopped task" annotation_1393508063:"Started task" annotation_1393510899:"Stopped task" annotation_1393943609:"Started task" annotation_1393944600:"Stopped task" annotation_1393953778:"Started task" annotation_1393955983:"Stopped task" annotation_1394006433:"Started task" annotation_1394008236:"Stopped task" description:"corriger la tropo" end:"1395135695" entry:"1392828493" modified:"1395135695" project:"aces" status:"completed" urgency:"H" uuid:"278e5d73-fb9c-49a5-b85c-b92fc488ed8b"]
{code}

It is the only line with "urgency" in it... I usually don't fiddle with the urgency settings for tasks and prefer to rely on due dates. But who knows, maybe I tried something on this one... I can't remember what, though.

So :
*  It looks weird to have 2 "urgency" fields in the "export" output.
*  This seems to be because there is one already in completed.data, and it shouldn't be here. I edited it out, now "export" only outputs 1 "urgency" field ("4") and json.loads() works.
*  So it appears that  it was my database that was corrupted for 1 year... It is just that the "export" output, until 2.4.1, masked this corruption.
*  I tried to write "task 26 mod urgency:H" to see if it was this command that corrupted the database, but there is a (welcomed) error message with the current version : "The value 'H' is not a valid numeric value.". This test has been added by commit https://git.tasktools.org/projects/TM/repos/task/commits/d2685a8212405b5a9fae8ce9a6f85e07c95f8872 so I probably was able to corrupt my database this way before the modification, and it shouldn't be possible anymore (well done, thanks !).

Thanks for your help !
